lib/chatops_deployer/app.rb

```diff
@@ -26,10 +26,12 @@ class App < Sinatra::Base
       json = JSON.parse(request.body.read)
       post_url = json['callback_url']
       clean = json['clean'] != 'false'
+      host = json['host'] != 'github.com'
```

Comment:
This would return `true` or `false` and store it in `host`. Is that expected? (It makes sense in the line above, but I'm not sure here)